runtime/doc/usr_90.txt

```diff
@@ -415,10 +415,8 @@ example for when "/usr/local" was used as the root: >
 
 	rm -rf /usr/local/share/vim/vim61
 	rm /usr/local/bin/vim
-	rm /usr/local/bin/vimtutor
 	rm /usr/local/bin/xxd
 	rm /usr/local/man/man1/vim.1
-	rm /usr/local/man/man1/vimtutor.1
```

Comment:
To keep this PR clean, I'll open one which cleans up `usr_90.txt`. Should be pretty quick I'd imagine.